task 39

Fargo02 opened the pull request:
В SearchFragment добавить обработку пустого состояния и ошибок

Fargo02 committed:
task-39
Изменил отступы RecyclerView по горизонтели

Fargo02 committed:
task-39
Добавил обработку пустого состояние в SearchFragment

Fargo02 committed:
task-39
Удалил метод setGroupVisibility, т.к. он эквивалентен visible().
Добавил плашку CountVacancies.

pyclan88 commented on app/src/main/java/ru/practicum/android/diploma/ui/search/SearchFragment.kt:
```diff
@@ -34,12 +38,29 @@ class SearchFragment : BindingFragment<FragmentSearchBinding>() {
     private fun render(state: VacancyState) {
         when (state.vacanciesList) {
-            is VacancyState.VacanciesList.Empty -> {}
+            is VacancyState.VacanciesList.Empty -> with(binding) {
```
не лучше будет вынести реализацию Empty стейта вынести в отдельный метод?

Fargo02 committed:
task-39
Вынес стейты в отдельное функции, добавил обработку стейта ошибка и нет интернета
